### DISCUSSION TITLE
Should WrapVariableVariableNameInCurlyBracesRector work on variable property names?

### DISCUSSION THREAD

**Discussion by aka-tpayne** [rectorphp/rector#6748] [General]
I've run this rule and it didn't apply to something like `$node->$field['name']` or `$node->$key`. Is it intended to work on that, or just some standalone variables?

**Comment by TomasVotruba** [comment]
Thank you for your report. This is rather an issue report. Could you send us failing test case for this rule with input and expected code in PR?

**Reply by aka-tpayne** [reply]
Absolutely. I'll file something.
